packages/sit-onyx/src/components/OnyxInput/OnyxInput.vue

```diff
@@ -129,6 +149,16 @@ const shouldShowCounter = computed(() => props.withCounter && props.maxlength);
 @use "../../styles/mixins/density.scss";
 @use "../../styles/mixins/input.scss";
 
+.onyx-use-optional:not(:has(.onyx-required-marker)) {
+  .onyx-input__optional {
+    display: inline-block;
+  }
+}
+
+.onyx-info-tooltip {
+  margin-left: var(--onyx-spacing-2xs);
+}
```

Comment:
This will apply the margin globally to all info tooltips, we should move it inside the `.onyx-input` and `.onyx-textarea`